packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -114,6 +116,19 @@ export const UIModeView: React.FC<{}> = ({
     });
   }, []);
 
+  const configProjects = React.useMemo(() => {
```

Comment:
Place this code next to where testModel is created or first used, it does not look like we need the memo indirection there. I actually think it was where it should be, why extracting?